www/install.rst

```diff
@@ -79,14 +75,32 @@ popular package managers you can install.
 Macports
 ~~~~~~~~
 
-To install the scipy stack for Python 2.7 with `Macports
+To install the SciPy stack for Python 2.7 with `Macports
 <http://www.macports.org>`_ execute this command in a terminal::
 
     sudo port install py27-numpy py27-scipy py27-matplotlib py27-ipython +notebook py27-pandas py27-sympy py27-nose
 
+Homebrew
+~~~~~~~~
```

Comment:
Maybe better to replace these 3rd level section headers by just boldface `**Homebrew**`.
This page should preferably be kept as short as possible, and the 3rd level headers start to make it look a bit messy.